task/buildah-min/0.4/buildah-min.yaml

```diff
@@ -367,7 +367,28 @@ spec:
         BUILD_ARG_FLAGS+=("--build-arg=$build_arg")
       done
 
-      dockerfile-json "${BUILD_ARG_FLAGS[@]}" "$dockerfile_copy" > /shared/parsed_dockerfile.json
+      # Dockerfile-json cannot parse Buildah's host variables, we have to pass them manually
+      BUILDAH_INFO=$(buildah info)
+      BUILDAH_OS=$(jq '.host.os' <<< "$BUILDAH_INFO" | tr -d '"' | tr -d "'")
+      BUILDAH_ARCH=$(jq '.host.arch' <<< "$BUILDAH_INFO" | tr -d '"' | tr -d "'")
+      BUILDAH_VARIANT=$(jq '.host.variant' <<< "$BUILDAH_INFO"  | tr -d '"' | tr -d "'")
```

Comment:
Nitpick: since the intent is to get raw strings, a more obvious way to do that is `jq`'s `-r/--raw-output` flag rather than the `tr -d '"'` calls

Reply:
Good suggestion, applied.